daemon/src/collab_settlement_maker.rs

```diff
@@ -18,7 +17,7 @@ use xtra_productivity::xtra_productivity;
 pub struct Actor {
     cfd: Cfd,
     projection: xtra::Address<projection::Actor>,
-    on_completed: Box<dyn MessageChannel<Completed<CollaborativeSettlement>>>,
+    on_completed: Box<dyn MessageChannel<CollaborativeSettlementCompleted>>,
```

Comment:
Note: I think the creation of the alias is somehow not part of this commit although it should be. This just shows the usage of the alias but not the alias itself.

Sidenote: 
We had this alias before, but I thought we agreed that we don't need to introduce one for collaborative settlement 😅

Reply:
I reintroduced it because adding the error type everywhere was annoying :D

Reply:
Thanks for pointing out the messy commit, will fix!